data/nlu/nlu.md

```diff
@@ -3466,6 +3479,8 @@
 - you're the most moronic person I know
 - your mom gay
 - you are annoying
+- don't you have a sweater
```

Comment:
@TyDunn is this actually an insult? 😂

Reply:
PTSD from people always calling me out for "not being dressed properly"

Reply:
i think OOS for that